Source/Settings.as

```diff
@@ -116,27 +116,35 @@ vec4 Setting_Gamepad_ClassicUpColor = vec4(0.2f, 1, 0.6f, 1);
 [Setting category="Gamepad" name="Classic down color" color if="Setting_Gamepad_Style Classic"]
 vec4 Setting_Gamepad_ClassicDownColor = vec4(1, 0.6f, 0.2f, 1);
 
-[Setting category="Gamepad" name="Classic off alpha" drag min=0 max=1 if="Setting_Gamepad_Style Classic"]
+[Setting category="Gamepad" name="Inactive alpha" min=0 max=1]
 float Setting_Gamepad_OffAlpha = 0.33f;
 
 [Setting category="Gamepad" name="Display up/down arrow symbols" if="Setting_Gamepad_Style Uniform"]
 bool Setting_Gamepad_UpDownSymbols = true;
 
-[Setting category="Gamepad" name="Cateye use simple steer" if="Setting_Gamepad_Style Cateye"]
+[Setting category="Gamepad" name="Cateye use simple steer" description="Does not yet show fill gradient correctly" if="Setting_Gamepad_Style Cateye"]
 bool Setting_Gamepad_CateyeUseSimpleSteer = false;
 
-[Setting category="Gamepad" name="Display steer percentage" if="Setting_Gamepad_Style Uniform"]
+[Setting category="Gamepad" name="Display steer percentage"]
 bool Setting_Gamepad_SteerPercentage = false;
 
+[Setting category="Gamepad" name="Display steer percentage symbol"]
+bool Setting_Gamepad_SteerPercentageSymbol = true;
+
+[Setting category="Gamepad" name="Steer percentage spacing" min=-100 max=100]
+float Setting_Gamepad_SteerPercentageSpacing = 0.0f;
+
+[Setting category="Gamepad" name="Font"]
+string Setting_Gamepad_Font = "DroidSans.ttf";
+
 [Setting
 	category="Gamepad"
-	name="Steer percentage size"
-	drag min=2 max=40
-	if="Setting_Gamepad_Style Uniform"]
-int Setting_Gamepad_SteerPercentageSize = 16;
+	name="Font size"
+	min=2 max=40]
+int Setting_Gamepad_FontSize = 16;
 
-[Setting category="Gamepad" name="Text and symbol color" color if="Setting_Gamepad_Style Uniform"]
-vec4 Setting_Gamepad_TextColor = vec4(1, 1, 1, 1);
+[Setting category="Gamepad" name="Font color" color]
+vec4 Setting_Gamepad_FontColor = vec4(1, 1, 1, 1);
```

Comment:
This breaks settings backwards compatibility; you should revert the variable name change.

We also never use the term `Font color` anywhere, just `Text color`.